pages/ref/api-strings.mdx

```diff
@@ -0,0 +1,348 @@
+let fizz: StringBuilder = beginString();
+let buzz: String = fizz.toString();
+```
+
+## StringBuilder.toCell
+
+```tact
+extends fun toCell(self: StringBuilder): Cell;
+```
+
+Extension function for the [`StringBuilder{:tact}`][p].
+
+Returns an assembled [`Cell{:tact}`][p] from a [`StringBuilder{:tact}`][p].
+
+Usage example:
+
+```tact
+let fizz: StringBuilder = beginString();
+let buzz: Cell = fizz.toCell();
+```
+
+## StringBuilder.toSlice
+
+```tact
+extends fun toSlice(self: StringBuilder): Slice;
+```
+
+Extension function for the [`StringBuilder{:tact}`][p].
+
+Returns an assembled [`Cell{:tact}`][p] as a [`Slice{:tact}`][p] from a [`StringBuilder{:tact}`][p]. Alias to [`self.toCell().asSlice(){:tact}`](/ref/api-cells#cellasslice).
+
+Usage example:
+
+```tact
+let s: StringBuilder = beginString();
+let fizz: Slice = s.toSlice();
+let buzz: Slice = s.toCell().asSlice();
+
+fizz == buzz; // true
+```
+
+## String.asSlice
+
+```tact
+extends fun asSlice(self: String): Slice;
+```
+
+Extension function for the [`String{:tact}`][p].
+
+Returns a [`Slice{:tact}`][p] from a [`String{:tact}`][p] by trying to pack all of its bits into a continuous list of [Cells][p], each referencing the next one and opening them all for future parsing.
+
+Note, that there's no indication of how many bytes a particular character could take in the [`Slice{:tact}`][p] or how deep the list of references is going to be, so use this function only if you know what you're doing.
+
+Usage example:
+
+```tact
+let s: String = "It's alive! It's alive!!!";
+let fizz: Slice = s.asSlice();
+let buzz: Slice = s.asSlice().asString().asSlice();
+
+fizz == buzz; // true, but be careful as it's not always the case
+```
+
+<Callout>
+
+  See how `String.asSlice{:tact}` function can be used in practice: [How to convert a `String` to an `Int`](/cookbook/type-conversion#how-to-convert-a-string-to-an-int).
+
+</Callout>
+
+## String.asComment
+
+```tact
+extends fun asComment(self: String): Cell;
+```
+
+Extension function for the [`String{:tact}`][p].
+
+Returns a [`Cell{:tact}`][p] from a [`String{:tact}`][p] by prefixing the latter with four null bytes. This format is used for passing text comments as message bodies.
+
+Usage example:
+
+```tact
+let s: String = "When life gives you lemons, call them 'yellow oranges' and sell them for double the price.";
+let fizz: Cell = s.asComment();
+
+let b: StringBuilder = beginComment();
+b.append(s);
+let buzz: Cell = b.toCell();
+
+fizz == buzz; // true
+```
+
+## String.fromBase64
+
+```tact
+extends fun fromBase64(self: String): Slice;
+```
+
+Extension function for the [`String{:tact}`][p].
+
+Returns a [`Slice{:tact}`][p] out of the decoded [Base64](https://en.wikipedia.org/wiki/Base64) [`String{:tact}`][p]. Alias to `self.asSlice().fromBase64(){:tact}`.
+
+Note, that this function is very limited and can be used only for small strings that fit in a single [`Cell{:tact}`][p] (i.e., strings that contain no more than 1023 bits of data).
```

Comment:
Do we get a compile-time error when it obviously does not fit?

Reply:
Nope, we don't: https://github.com/tact-lang/tact-docs/pull/206/commits/92dcd713aa1abae5ae84d3b7d2756bd18bd71599